taxi/models.py

```diff
@@ -1,6 +1,7 @@
 from django.db import models
 from django.contrib.auth.models import AbstractUser
 from django.urls import reverse
+from .validators import LicenseValidator
```

Comment:
Use absolute imports only and add blank line between different groups of imports according to PEP8